app/javascript/packs/admin/users/memberIndex.js

```diff
@@ -105,3 +106,24 @@ const handleEmailCopy = (copyEmail) => {
       );
     });
 };
+
+// The reason that we loop through the elements is because we have alternate layouts
+// for different screen sizes
+document.querySelectorAll('.js-export-csv-modal-trigger').forEach((item) => {
+  item.addEventListener('click', () => {
+    showWindowModal({
+      title: 'Download Member Data',
+      contentSelector: '#export-csv-modal',
+      overlay: true,
+      onOpen: () => {
+        document
+          .querySelectorAll('#window-modal .js-export-csv-modal-cancel')
+          .forEach((item) => {
+            item.addEventListener('click', () => {
+              closeWindowModal();
+            });
+          });
```

Comment:
Because we're targeting specifically the export button inside the current modal (`'#window-modal .js-export-csv-modal-cancel'`), there should only ever be one of these. So it should be safe to do e.g.

```suggestion
          document
          .querySelector('#window-modal .js-export-csv-modal-cancel')
          ?.addEventListener('click', closeWindowModal);
```